server/tests/test_inbound_phase2_standalone.py

```
@@ -0,0 +1,112 @@
+        else:
+            print("❌ FAILED: is_callback incorrect")
+            
+        if vars.get("name") == "John":
+            print("✅ SUCCESS: Customer name extracted")
+        else:
+            print(f"❌ FAILED: Customer name incorrect. Got: {vars.get('name')}")
+            
+        if overrides.get("assistantId") == "inbound-assistant-id-123":
+            print("✅ SUCCESS: Switched to Inbound Assistant")
+        else:
+             print(f"❌ FAILED: Did not switch assistant")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)